libs/styles/src/lib/scss/components/_steps.scss

```diff
@@ -76,6 +76,12 @@
         flex-direction: column;
       }
       @include cvi-breakpoint-up(sm) {
+        #{$base}.is-scrollable & {
+          display: flex;
+          padding-left: get-spacing('loksa');
```

Comment:
Added spacing, as when navigation is scrollable it feels like edges are too close to screen (so first and last button edges are little bit like cut off). Maybe it's me, but with testing, it felt better to add 4px spacing.